roles/redhat-subscriptions/tests/test.yml

```diff
@@ -0,0 +1,4 @@
+---
+- hosts: sat63
+  roles:
+    - partition-disk
```

Comment:
Should the tests be using the redhat-subscriptions role?

Reply:
I missed this, a typo, will update it. I was more looking into the directory structure and overlooked this.